src/util/taproot.rs

```diff
@@ -790,16 +827,16 @@ impl LeafVersion {
         match version {
             TAPROOT_LEAF_TAPSCRIPT => Ok(LeafVersion::TapScript),
             TAPROOT_ANNEX_PREFIX => Err(TaprootError::InvalidTaprootLeafVersion(TAPROOT_ANNEX_PREFIX)),
-            odd if odd & TAPROOT_LEAF_MASK != odd => Err(TaprootError::InvalidTaprootLeafVersion(odd)),
-            future => Ok(LeafVersion::Future(future)),
+            0xC1 => Err(TaprootError::InvalidTaprootLeafVersion(0xC1)),
```

Comment:
Any special reason why this needs a special line? I think this would be handled by the `future` match arm in the next line.

Reply:
Fixed